tests/test_tocdirective/test_toc_latex_urllink.tex

```diff
@@ -54,14 +31,9 @@ \chapter{Subfolder index}
 \label{\detokenize{subfolder/index:subfolder-index}}\label{\detokenize{subfolder/index::doc}}
 \sphinxAtStartPar
 \sphinxstylestrong{Table of contents directive}
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{subfolder/asubpage::doc}]{\sphinxcrossref{Asubpage}}}
-
-\end{itemize}
 
 
-\chapter{A subpage}
+\section{A subpage}
```

Comment:
here (with `jupyterbook-latex==0.2.0`) the header from a sub-toctrre is incorrectly assigned as a `section`

Reply:
but here actually `jupyterbook-latex` should probably override `latex_toplevel_sectioning` to `section`, i.e.:

- tocs with format `jb-article`: set `latex_toplevel_sectioning = "section"` and don't convert toctree captions to headers
- tocs with format `jb-book` and no parts (i.e. only a single top-level toctree): set `latex_toplevel_sectioning = "chapter"` and don't convert toctree captions to headers
- tocs with format `jb-book` and parts (i.e. multiple top-level toctrees): set `latex_toplevel_sectioning = "part"` and do convert toctree captions to headers

Reply:
done